lib/shared/addon/mixins/node-driver.js

```diff
@@ -92,6 +93,16 @@ export default Mixin.create(NewOrEdit, ManageLabels, {
     this._super(...arguments);
 
     if ( !get(this, 'editing') && typeof get(this, 'bootstrap') === 'function') {
```

Comment:
Why can't this be in `bootstrap()` of the aliyun driver instead of in the middle of generic driver?

Reply:
@vincent99 yes, it can be. I have moved it into aliyun driver. Thanks